src/test/java/net/solmis/cityparking/FakeDatabaseServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package net.solmis.cityparking;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FakeDatabaseServiceTest {
```

Comment:
testy pod obiekt będący fakiem?

Reply:
Ze względu na sposób, w jaki to zrobiłem (FakeDatabaseService), myślę, że takie testy mogą być przydatne. Ta klasa jednak nie jest najlepiej zaprojektowana, co przyznałem już w innym komentarzu.